src/controllers/costTime.controller.js

```diff
@@ -1,21 +1,26 @@
 'use strict'
 
-const { WASTE_IN_DEPOSIT_FOR_RECOVERY } = require('../constants').PermitTypes.STANDARD_RULES
+const Constants = require('../constants')
+const { WASTE_IN_DEPOSIT_FOR_RECOVERY } = Constants.PermitTypes.STANDARD_RULES
+const { MCP_CATEGORIES_PROCESSING_TIME, DEFAULT_CATEGORIES_PROCESSING_TIME } = Constants.PROCESSING_TIME
 const Routes = require('../routes')
 const BaseController = require('./base.controller')
 const CostTime = require('../models/taskList/costTime.task')
 const RecoveryService = require('../services/recovery.service')
+const StandardRuleType = require('../persistence/entities/standardRuleType.entity')
 
 module.exports = class CostTimeController extends BaseController {
   async doGet (request, h) {
     const pageContext = this.createPageContext(h)
     const context = await RecoveryService.createApplicationContext(h, { applicationLine: true, standardRule: true })
     const { applicationLine = {}, standardRule = {} } = context
+    const { categoryName } = await StandardRuleType.getById(context, standardRule.standardRuleTypeId)
 
     // Default to 0 when the balance hasn't been set
     const { value = 0 } = applicationLine
 
     pageContext.cost = value.toLocaleString()
+    pageContext.time = categoryName.includes('MCP') ? MCP_CATEGORIES_PROCESSING_TIME : DEFAULT_CATEGORIES_PROCESSING_TIME
```

Comment:
Not specifically a problem here but the functions in the StandardRuleType class and the other places where it is being used are converting the categoryName to lower-case (for reasons that escape me), so it's theoretically possible that this check could fail if that class overrode getById at some point in order to be consistent.
So maybe just do a case-insensitive comparison?

Reply:
I am now testing for an exact (to lower case) match of the categoryName to either 'mcpd-sg' or 'mcpd-mcp' giving a time of 9 weeks.  The default is 13 weeks.